src/main/java/org/isf/security/jwt/TokenProvider.java

```diff
@@ -83,11 +83,11 @@ public void init() {
 		byte[] keyBytes = secret.getBytes(StandardCharsets.UTF_8);
 		this.key = Keys.hmacShaKeyFor(keyBytes);
 
-		// 30 minutes (900,000 milliseconds)
-		this.tokenValidityInMilliseconds = 1000L * 60 * 30;
+		// default 30 minutes (900,000 milliseconds)
+		this.tokenValidityInMilliseconds = env.getProperty("jwt.token.validityInMilliseconds", Long.class, 1000L * 60 * 30);
 
-		// 3 days (604,800,000 milliseconds)
-		this.tokenValidityInMillisecondsForRememberMe = 1000L * 60 * 60 * 24 * 3;
+		// default 3 days (604,800,000 milliseconds)
+		this.tokenValidityInMillisecondsForRememberMe = env.getProperty("jwt.token.validityInMillisecondsForRememberMe", Long.class, 1000L * 60 * 60 * 24 * 3);
```

Comment:
shall we put a `LOGGER.debug` of the read properties as done in line 82?

Reply:
(to be honest I don't like even the one in line 82 unless for development purpose)

Reply:
done! 👍🏼